Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabSystemScriptingHandler.cpp

```diff
@@ -73,12 +74,24 @@ namespace AzToolsFramework::Prefab
         AzToolsFramework::ToolsApplicationRequestBus::BroadcastResult(result, &AzToolsFramework::ToolsApplicationRequestBus::Events::FindCommonRootInactive,
             entities, commonRoot, &topLevelEntities);
 
+
         auto containerEntity = AZStd::make_unique<AZ::Entity>();
-        containerEntity->CreateComponent<Components::TransformComponent>();
+
```

Comment:
While default overrides are only on the transform component, when overriding support is added later, a user can place an override on any component. So, the other components should also use the hashing mechanism like the transform component.

Reply:
Agreed, but I'd like to deal with that when we come to it.

For components further down, we're going to have to add some additional info to the hash, since they obviously can't all be a hash of the file name.  `{FileName}_{ComponentType}` might work for most of them, but I expect there are some components you can double up on, so they'll end up with duplicates.  Maybe we can count them up and assign each a number?  I'm not sure how stable that will be.